dockerfiles/cli/tests/cmd_start_stop_restart_tests.bats

```diff
@@ -71,18 +68,18 @@ teardown() {
     [ "$output" = "Default port 8080 for che server is used. Cannot run this test on default che server port" ]
   fi
   tmp_path="${TESTRUN_DIR}"/cli_cmd_restart_with_default_settings
-  echo $tmp_path
   mkdir -p "${tmp_path}"
   docker run --rm -v "${SCRIPTS_DIR}":/scripts/base -v /var/run/docker.sock:/var/run/docker.sock -v "${tmp_path}":/data -e CHE_CONTAINER=chetest $CLI_IMAGE start --skip:nightly --skip:pull
   [[ "$(docker inspect --format='{{.State.Running}}' chetest)" == "true" ]]
   ip_address=$(docker inspect -f {{.NetworkSettings.Networks.bridge.IPAddress}} chetest)
   curl -fsS http://${ip_address}:8080  > /dev/null
+  che_container_id=$(docker inspect --format="{{.Id}}" chetest)
 
   #WHEN
   docker run --rm -v "${SCRIPTS_DIR}":/scripts/base -v /var/run/docker.sock:/var/run/docker.sock -v "${tmp_path}":/data -e CHE_CONTAINER=chetest $CLI_IMAGE restart --skip:nightly --skip:pull
 
   #THEN
-  [[ "$(docker inspect --format='{{.State.Running}}' chetest)" == "true" ]]
+  [[ "$(docker inspect --format="{{.Id}}" chetest)" != "$che_container_id" ]]
```

Comment:
Does the restart command on docker CLI container delete a container and start a new one? If not and it just reuse `docker restart` with some more magic, then the container ID is still the same.